staging/src/k8s.io/api/scheduling/v1beta1/types.go

```diff
@@ -24,6 +24,10 @@ import (
 // +genclient
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:prerelease-lifecycle-gen:introduced=1.11
+// +k8s:prerelease-lifecycle-gen:deprecated=1.14
+// +k8s:prerelease-lifecycle-gen:removed=1.22
```

Comment:
One qq: what should scheduling team do in 1.22? Remove the v1beta1 Priority API in the code, and then run hack/xyz to update the tags? Or something else.

Reply:
> One qq: what should scheduling team do in 1.22? Remove the v1beta1 Priority API in the code, and then run hack/xyz to update the tags? Or something else.

All clients and controllers should be updated to use the v1 level of the API.

In 1.22, the REST handler code will automatically stop serving APIs based on these tags, so no other action should be required.

Reply:
SG. Thanks for explanation.